src/main/java/Deadline.java

```diff
@@ -0,0 +1,46 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
+
+public class Deadline extends Task {
+    private String dueDate;
+    private String dueDateInSaveFormat;
+    private Deadline(String name, String dueDate, String dueDateInSaveFormat) {
+        super(name);
+        this.dueDate = dueDate;
+        this.dueDateInSaveFormat = dueDateInSaveFormat;
+    }
+
+    public static Deadline createDeadline(String name, String dueDate) throws DukeException{
```

Comment:
Should there be a space after DukeException? I've noticed this issue in several other places too.